PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -285,4 +293,34 @@ class PsfPowershellScriptRunner
 
         return true;
     }
+
+    void MakeAttributeList(LPPROC_THREAD_ATTRIBUTE_LIST& attributeList)

```

Comment:
>void [](start = 4, length = 4)

Should return the LPPROC_THREAD_ATTRIBUTE_LIST (or RAII equivalent) rather than using an out param.